src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -201,12 +201,21 @@ FullyConnected_bf_tiled::GetAutoTuneParams(const fully_connected_params& params,
         max_tile_ofm *= 2;
 
     if (params.is_shape_agnostic) {
+        // Use special tuning params for Gen12HP dGPUs, since these parameters demonstrate higher performance
+        // due to better HW utilization (reduced TILE_OFM parameter) and better assembler kernel's code
+        // generation (extended TILE_K parameter) for both FP16 and FP32 data types
         if (dtype == Datatype::F16) {
             // tune_params(tile_b, tile_ofm, tile_ifm, tile_k, dispatch_bsv, dispatch_fsv, exec_options)
-            selector.Case(tune_params(8,  std::min(max_tile_ofm, 2u), 1, 2, 1,  1, EXE_MODE_AGE_BASED));
+            if (params.engineInfo.supports_immad)
+                selector.Case(tune_params(8, 1, 1, 4, 1, 1, EXE_MODE_AGE_BASED));
```

Comment:
Is this trend same for various shapes?

Reply:
Yes, I have checked this for several shapes and both FP16 and FP32 precisions from different bert models (including bert-large, bert-base and some other)